pyam/core.py

```diff
@@ -8,6 +8,10 @@
 import numpy as np
 import pandas as pd
 
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from datapackage import Package
```

Comment:
let's make this optional with:

```
try:
    from datapackage import Package
    HAS_DATAPACKAGE = True
except IoError:
    HAS_DATAPACKAGE = False
```

and then where datapackage is used either have a decorator (like we do in tests) or an explicit check for `HAS_DATAPACKAGE`

Reply:
done